WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:17 [INFO ] Loaded 0 jobs.
SKIP: Extension "image-gradients" is not installed
-------------------- >> begin captured logging << --------------------
engineio: INFO: Server initialized for gevent.
digits: INFO: Loaded 0 jobs.
--------------------- >> end captured logging << ---------------------
SKIP: Extension "image-gradients" is not installed
-------------------- >> begin captured logging << --------------------
engineio: INFO: Server initialized for gevent.
digits: INFO: Loaded 0 jobs.
--------------------- >> end captured logging << ---------------------
digits.dataset.generic.test_views.TestImageGradientViews.test_nonexistent_dataset ... ok
digits.dataset.generic.test_views.TestImageGradientViews.test_page_dataset_new ... FAIL

<html>
<head>
<link rel="icon" href="/static/images/nvidia.ico" />
<link rel="stylesheet" href="/static/css/bootstrap.min.css?ver=77b0f4b9a9f48665f738eb0ae1624192">
<link rel="stylesheet" href="/static/css/bootstrap-theme.min.css?ver=77b0f4b9a9f48665f738eb0ae1624192">
</head>

<body>
<h2>ValueError</h2>
<p class="text-danger">Unknown extension &#39;image-gradients&#39;</p>
<pre class="text-muted">
Traceback (most recent call last):
  File "/apps/python/2.7.11/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/apps/python/2.7.11/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/data/zim021/digits-proj/DIGITS/digits/webapp.py", line 81, in decorated
    return f(*args, **kwargs)
  File "/data/zim021/digits-proj/DIGITS/digits/utils/auth.py", line 55, in decorated
    return f(*args, **kwargs)
  File "/data/zim021/digits-proj/DIGITS/digits/dataset/generic/views.py", line 44, in new
    raise ValueError("Unknown extension '%s'" % extension_id)
ValueError: Unknown extension 'image-gradients'

</pre>

</body>
</html>
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] Create train_db DB task started.
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] <class 'digits.dataset.tasks.create_generic_db.CreateGenericDbTask'>
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111118-6896 there: True True
Running in slurm mode
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_generic_db.py 20161209-111118-6896 --stage=train_db --jobs_dir=/data/zim021/digits-proj/tmp/tmpfEJdl5"
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] Create val_db DB task started.
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] <class 'digits.dataset.tasks.create_generic_db.CreateGenericDbTask'>
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111118-6896 there: True True
Running in slurm mode
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_generic_db.py 20161209-111118-6896 --stage=val_db --jobs_dir=/data/zim021/digits-proj/tmp/tmpfEJdl5"
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] Create test_db DB task started.
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] <class 'digits.dataset.tasks.create_generic_db.CreateGenericDbTask'>
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111118-6896 there: True True
Running in slurm mode
2016-12-09 11:11:18 [20161209-111118-6896] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_generic_db.py 20161209-111118-6896 --stage=test_db --jobs_dir=/data/zim021/digits-proj/tmp/tmpfEJdl5"
2016-12-09 11:11:18 [20161209-111118-6896] [WARNING] Create train_db DB unrecognized output: salloc: Granted job allocation 4016030
2016-12-09 11:11:18 [20161209-111118-6896] [WARNING] Create val_db DB unrecognized output: salloc: Granted job allocation 4016031
2016-12-09 11:11:18 [20161209-111118-6896] [WARNING] Create test_db DB unrecognized output: salloc: Granted job allocation 4016032
2016-12-09 11:11:18 [20161209-111118-6896] [WARNING] Create test_db DB unrecognized output: srun: Job step created
2016-12-09 11:11:18 [20161209-111118-6896] [WARNING] Create val_db DB unrecognized output: srun: Job step created
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create train_db DB unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create train_db DB unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create test_db DB unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create test_db DB unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create test_db DB unrecognized output: 2016-12-09 11:11:21 [INFO ] Generic DB creation Done
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create val_db DB unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create val_db DB unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create test_db DB unrecognized output: salloc: Relinquishing job allocation 4016032
2016-12-09 11:11:21 [20161209-111118-6896] [INFO ] Create test_db DB task completed.
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create val_db DB unrecognized output: 2016-12-09 11:11:21 [INFO ] Generic DB creation Done
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create train_db DB unrecognized output: 2016-12-09 11:11:21 [INFO ] Generic DB creation Done
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create val_db DB unrecognized output: salloc: Relinquishing job allocation 4016031
2016-12-09 11:11:21 [20161209-111118-6896] [WARNING] Create train_db DB unrecognized output: salloc: Relinquishing job allocation 4016030
2016-12-09 11:11:21 [20161209-111118-6896] [INFO ] Create val_db DB task completed.
2016-12-09 11:11:21 [20161209-111118-6896] [INFO ] Create train_db DB task completed.
2016-12-09 11:11:21 [20161209-111118-6896] [INFO ] Job complete.
digits.dataset.generic.test_views.TestImageProcessingCreated.test_dataset_json ... ok
digits.dataset.generic.test_views.TestImageProcessingCreated.test_edit_name ... 2016-12-09 11:11:22 [20161209-111118-6896] [INFO ] Set name to "new name".
ok
digits.dataset.generic.test_views.TestImageProcessingCreated.test_edit_notes ... 2016-12-09 11:11:22 [20161209-111118-6896] [INFO ] Updated notes.
ok
digits.dataset.generic.test_views.TestImageProcessingCreated.test_entry_counts ... ok
digits.dataset.generic.test_views.TestImageProcessingCreated.test_explore_features ... ok
digits.dataset.generic.test_views.TestImageProcessingCreated.test_feature_dims ... ok
digits.dataset.generic.test_views.TestImageProcessingCreated.test_index_json ... ok
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] Create train_db DB task started.
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] <class 'digits.dataset.tasks.create_generic_db.CreateGenericDbTask'>
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111122-af40 there: True True
Running in slurm mode
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_generic_db.py 20161209-111122-af40 --stage=train_db --jobs_dir=/data/zim021/digits-proj/tmp/tmpfEJdl5"
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] Create val_db DB task started.
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] <class 'digits.dataset.tasks.create_generic_db.CreateGenericDbTask'>
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111122-af40 there: True True
Running in slurm mode
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_generic_db.py 20161209-111122-af40 --stage=val_db --jobs_dir=/data/zim021/digits-proj/tmp/tmpfEJdl5"
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] Create test_db DB task started.
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] <class 'digits.dataset.tasks.create_generic_db.CreateGenericDbTask'>
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111122-af40 there: True True
Running in slurm mode
2016-12-09 11:11:22 [20161209-111122-af40] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_generic_db.py 20161209-111122-af40 --stage=test_db --jobs_dir=/data/zim021/digits-proj/tmp/tmpfEJdl5"
2016-12-09 11:11:22 [20161209-111122-af40] [WARNING] Create train_db DB unrecognized output: salloc: Granted job allocation 4016033
2016-12-09 11:11:22 [20161209-111122-af40] [WARNING] Create val_db DB unrecognized output: salloc: Granted job allocation 4016034
2016-12-09 11:11:22 [20161209-111122-af40] [WARNING] Create test_db DB unrecognized output: salloc: Granted job allocation 4016035
2016-12-09 11:11:23 [20161209-111122-af40] [WARNING] Create test_db DB unrecognized output: srun: Job step created
2016-12-09 11:11:23 [20161209-111122-af40] [WARNING] Create val_db DB unrecognized output: srun: Job step created
2016-12-09 11:11:25 [20161209-111122-af40] [WARNING] Create train_db DB unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:25 [20161209-111122-af40] [WARNING] Create train_db DB unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:25 [20161209-111122-af40] [WARNING] Create train_db DB unrecognized output: 2016-12-09 11:11:25 [INFO ] Generic DB creation Done
2016-12-09 11:11:25 [20161209-111122-af40] [WARNING] Create test_db DB unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:25 [20161209-111122-af40] [WARNING] Create test_db DB unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:25 [20161209-111122-af40] [WARNING] Create test_db DB unrecognized output: 2016-12-09 11:11:25 [INFO ] Generic DB creation Done
2016-12-09 11:11:25 [20161209-111122-af40] [WARNING] Create train_db DB unrecognized output: salloc: Relinquishing job allocation 4016033
2016-12-09 11:11:25 [20161209-111122-af40] [WARNING] Create val_db DB unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:25 [20161209-111122-af40] [WARNING] Create val_db DB unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:25 [20161209-111122-af40] [INFO ] Create train_db DB task completed.
2016-12-09 11:11:26 [20161209-111122-af40] [WARNING] Create test_db DB unrecognized output: salloc: Relinquishing job allocation 4016035
2016-12-09 11:11:26 [20161209-111122-af40] [INFO ] Create test_db DB task completed.
2016-12-09 11:11:26 [20161209-111122-af40] [WARNING] Create val_db DB unrecognized output: 2016-12-09 11:11:26 [INFO ] Generic DB creation Done
2016-12-09 11:11:26 [20161209-111122-af40] [WARNING] Create val_db DB unrecognized output: salloc: Relinquishing job allocation 4016034
2016-12-09 11:11:26 [20161209-111122-af40] [INFO ] Create val_db DB task completed.
2016-12-09 11:11:26 [20161209-111122-af40] [INFO ] Job complete.
digits.dataset.generic.test_views.TestImageProcessingCreatedWithSeparateValidationDirs.test_dataset_json ... ok
digits.dataset.generic.test_views.TestImageProcessingCreatedWithSeparateValidationDirs.test_edit_name ... 2016-12-09 11:11:26 [20161209-111122-af40] [INFO ] Set name to "new name".
ok
digits.dataset.generic.test_views.TestImageProcessingCreatedWithSeparateValidationDirs.test_edit_notes ... 2016-12-09 11:11:26 [20161209-111122-af40] [INFO ] Updated notes.
ok
digits.dataset.generic.test_views.TestImageProcessingCreatedWithSeparateValidationDirs.test_entry_counts ... ok
digits.dataset.generic.test_views.TestImageProcessingCreatedWithSeparateValidationDirs.test_explore_features ... ok
digits.dataset.generic.test_views.TestImageProcessingCreatedWithSeparateValidationDirs.test_feature_dims ... ok
digits.dataset.generic.test_views.TestImageProcessingCreatedWithSeparateValidationDirs.test_index_json ... ok
2016-12-09 11:11:26 [20161209-111126-160d] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:11:26 [20161209-111126-160d] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:11:26 [20161209-111126-160d] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d there: True True
Running in slurm mode
2016-12-09 11:11:26 [20161209-111126-160d] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmpmXuM6u /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d/labels.txt --min=2 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d/val.txt --percent_val=25.0"
2016-12-09 11:11:26 [20161209-111126-160d] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016036
2016-12-09 11:11:27 [20161209-111126-160d] [WARNING] Parse Folder (train/val) unrecognized output: srun: Job step created
2016-12-09 11:11:29 [20161209-111126-160d] [WARNING] Parse Folder (train/val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:29 [20161209-111126-160d] [WARNING] Parse Folder (train/val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:29 [20161209-111126-160d] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016036
2016-12-09 11:11:29 [20161209-111126-160d] [INFO ] Parse Folder (train/val) task completed.
2016-12-09 11:11:29 [20161209-111126-160d] [INFO ] Create DB (train) task started.
2016-12-09 11:11:29 [20161209-111126-160d] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:29 [20161209-111126-160d] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d there: True True
Running in slurm mode
2016-12-09 11:11:29 [20161209-111126-160d] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d/train.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d/train_db 10 10 --backend=lmdb --channels=3 --resize_mode=squash --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d/mean.binaryproto --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d/mean.jpg --shuffle --encoding=png"
2016-12-09 11:11:29 [20161209-111126-160d] [INFO ] Create DB (val) task started.
2016-12-09 11:11:29 [20161209-111126-160d] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:29 [20161209-111126-160d] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d there: True True
Running in slurm mode
2016-12-09 11:11:29 [20161209-111126-160d] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d/val.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111126-160d/val_db 10 10 --backend=lmdb --channels=3 --resize_mode=squash --shuffle --encoding=png"
2016-12-09 11:11:29 [20161209-111126-160d] [WARNING] Create DB (train) unrecognized output: salloc: Granted job allocation 4016037
2016-12-09 11:11:29 [20161209-111126-160d] [WARNING] Create DB (val) unrecognized output: salloc: Granted job allocation 4016038
2016-12-09 11:11:29 [20161209-111126-160d] [WARNING] Create DB (train) unrecognized output: srun: Job step created
2016-12-09 11:11:29 [20161209-111126-160d] [WARNING] Create DB (val) unrecognized output: srun: Job step created
2016-12-09 11:11:32 [20161209-111126-160d] [WARNING] Create DB (train) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:32 [20161209-111126-160d] [WARNING] Create DB (train) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:32 [20161209-111126-160d] [WARNING] Create DB (val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:32 [20161209-111126-160d] [WARNING] Create DB (val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:32 [20161209-111126-160d] [DEBUG] 24 images written to database
2016-12-09 11:11:32 [20161209-111126-160d] [WARNING] Create DB (train) unrecognized output: salloc: Relinquishing job allocation 4016037
2016-12-09 11:11:32 [20161209-111126-160d] [INFO ] Create DB (train) task completed.
2016-12-09 11:11:32 [20161209-111126-160d] [DEBUG] 6 images written to database
2016-12-09 11:11:32 [20161209-111126-160d] [WARNING] Create DB (val) unrecognized output: salloc: Relinquishing job allocation 4016038
2016-12-09 11:11:32 [20161209-111126-160d] [INFO ] Create DB (val) task completed.
2016-12-09 11:11:32 [20161209-111126-160d] [INFO ] Job complete.
digits.dataset.images.classification.test_views.TestCreated.test_backend_selection ... ok
digits.dataset.images.classification.test_views.TestCreated.test_clone ... 2016-12-09 11:11:32 [20161209-111132-f919] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:11:32 [20161209-111132-f919] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:11:32 [20161209-111132-f919] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919 there: True True
Running in slurm mode
2016-12-09 11:11:32 [20161209-111132-f919] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmpmXuM6u /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919/labels.txt --min=1 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919/val.txt --percent_val=25.0 --max=3"
2016-12-09 11:11:33 [20161209-111132-f919] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016039
2016-12-09 11:11:33 [20161209-111132-f919] [WARNING] Parse Folder (train/val) unrecognized output: srun: Job step created
2016-12-09 11:11:35 [20161209-111132-f919] [WARNING] Parse Folder (train/val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:35 [20161209-111132-f919] [WARNING] Parse Folder (train/val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:35 [20161209-111132-f919] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:35 [20161209-111132-f919] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:35 [20161209-111132-f919] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:35 [20161209-111132-f919] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016039
2016-12-09 11:11:35 [20161209-111132-f919] [INFO ] Parse Folder (train/val) task completed.
2016-12-09 11:11:35 [20161209-111132-f919] [INFO ] Create DB (train) task started.
2016-12-09 11:11:35 [20161209-111132-f919] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:35 [20161209-111132-f919] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919 there: True True
Running in slurm mode
2016-12-09 11:11:35 [20161209-111132-f919] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919/train.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919/train_db 10 10 --backend=lmdb --channels=3 --resize_mode=half_crop --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919/mean.binaryproto --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919/mean.jpg --shuffle --encoding=png"
2016-12-09 11:11:35 [20161209-111132-f919] [INFO ] Create DB (val) task started.
2016-12-09 11:11:35 [20161209-111132-f919] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:35 [20161209-111132-f919] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919 there: True True
Running in slurm mode
2016-12-09 11:11:35 [20161209-111132-f919] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919/val.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111132-f919/val_db 10 10 --backend=lmdb --channels=3 --resize_mode=half_crop --shuffle --encoding=png"
2016-12-09 11:11:35 [20161209-111132-f919] [WARNING] Create DB (train) unrecognized output: salloc: Granted job allocation 4016040
2016-12-09 11:11:35 [20161209-111132-f919] [WARNING] Create DB (val) unrecognized output: salloc: Granted job allocation 4016041
2016-12-09 11:11:35 [20161209-111132-f919] [WARNING] Create DB (train) unrecognized output: srun: Job step created
2016-12-09 11:11:35 [20161209-111132-f919] [WARNING] Create DB (val) unrecognized output: srun: Job step created
2016-12-09 11:11:38 [20161209-111132-f919] [WARNING] Create DB (train) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:38 [20161209-111132-f919] [WARNING] Create DB (train) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:38 [20161209-111132-f919] [DEBUG] 6 images written to database
2016-12-09 11:11:38 [20161209-111132-f919] [WARNING] Create DB (val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:38 [20161209-111132-f919] [WARNING] Create DB (val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:38 [20161209-111132-f919] [WARNING] Create DB (train) unrecognized output: salloc: Relinquishing job allocation 4016040
2016-12-09 11:11:38 [20161209-111132-f919] [INFO ] Create DB (train) task completed.
2016-12-09 11:11:38 [20161209-111132-f919] [DEBUG] 3 images written to database
2016-12-09 11:11:38 [20161209-111132-f919] [WARNING] Create DB (val) unrecognized output: salloc: Relinquishing job allocation 4016041
2016-12-09 11:11:38 [20161209-111132-f919] [INFO ] Create DB (val) task completed.
2016-12-09 11:11:38 [20161209-111132-f919] [INFO ] Job complete.
2016-12-09 11:11:39 [20161209-111138-1645] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:11:39 [20161209-111138-1645] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:11:39 [20161209-111138-1645] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645 there: True True
Running in slurm mode
2016-12-09 11:11:39 [20161209-111138-1645] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmpmXuM6u /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645/labels.txt --min=1 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645/val.txt --percent_val=25.0 --max=3"
2016-12-09 11:11:39 [20161209-111138-1645] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016042
2016-12-09 11:11:39 [20161209-111138-1645] [WARNING] Parse Folder (train/val) unrecognized output: srun: Job step created
2016-12-09 11:11:41 [20161209-111138-1645] [WARNING] Parse Folder (train/val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:41 [20161209-111138-1645] [WARNING] Parse Folder (train/val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:41 [20161209-111138-1645] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:41 [20161209-111138-1645] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:41 [20161209-111138-1645] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:41 [20161209-111138-1645] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016042
2016-12-09 11:11:41 [20161209-111138-1645] [INFO ] Parse Folder (train/val) task completed.
2016-12-09 11:11:41 [20161209-111138-1645] [INFO ] Create DB (train) task started.
2016-12-09 11:11:41 [20161209-111138-1645] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:41 [20161209-111138-1645] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645 there: True True
Running in slurm mode
2016-12-09 11:11:41 [20161209-111138-1645] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645/train.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645/train_db 10 10 --backend=lmdb --channels=3 --resize_mode=half_crop --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645/mean.binaryproto --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645/mean.jpg --shuffle --encoding=png"
2016-12-09 11:11:41 [20161209-111138-1645] [INFO ] Create DB (val) task started.
2016-12-09 11:11:41 [20161209-111138-1645] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:41 [20161209-111138-1645] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645 there: True True
Running in slurm mode
2016-12-09 11:11:41 [20161209-111138-1645] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645/val.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111138-1645/val_db 10 10 --backend=lmdb --channels=3 --resize_mode=half_crop --shuffle --encoding=png"
2016-12-09 11:11:41 [20161209-111138-1645] [WARNING] Create DB (train) unrecognized output: salloc: Granted job allocation 4016043
2016-12-09 11:11:41 [20161209-111138-1645] [WARNING] Create DB (val) unrecognized output: salloc: Granted job allocation 4016044
2016-12-09 11:11:41 [20161209-111138-1645] [WARNING] Create DB (train) unrecognized output: srun: Job step created
2016-12-09 11:11:41 [20161209-111138-1645] [WARNING] Create DB (val) unrecognized output: srun: Job step created
2016-12-09 11:11:44 [20161209-111138-1645] [WARNING] Create DB (train) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:44 [20161209-111138-1645] [WARNING] Create DB (train) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:44 [20161209-111138-1645] [WARNING] Create DB (val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:44 [20161209-111138-1645] [WARNING] Create DB (val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:44 [20161209-111138-1645] [DEBUG] 6 images written to database
2016-12-09 11:11:44 [20161209-111138-1645] [DEBUG] 3 images written to database
2016-12-09 11:11:44 [20161209-111138-1645] [WARNING] Create DB (train) unrecognized output: salloc: Relinquishing job allocation 4016043
2016-12-09 11:11:44 [20161209-111138-1645] [INFO ] Create DB (train) task completed.
2016-12-09 11:11:44 [20161209-111138-1645] [WARNING] Create DB (val) unrecognized output: salloc: Relinquishing job allocation 4016044
2016-12-09 11:11:44 [20161209-111138-1645] [INFO ] Create DB (val) task completed.
2016-12-09 11:11:44 [20161209-111138-1645] [INFO ] Job complete.
ok
digits.dataset.images.classification.test_views.TestCreated.test_dataset_json ... ok
digits.dataset.images.classification.test_views.TestCreated.test_edit_name ... 2016-12-09 11:11:45 [20161209-111126-160d] [INFO ] Set name to "new name".
ok
digits.dataset.images.classification.test_views.TestCreated.test_edit_notes ... 2016-12-09 11:11:45 [20161209-111126-160d] [INFO ] Updated notes.
ok
digits.dataset.images.classification.test_views.TestCreated.test_explore_train ... ok
digits.dataset.images.classification.test_views.TestCreated.test_explore_val ... ok
digits.dataset.images.classification.test_views.TestCreated.test_index_json ... ok
digits.dataset.images.classification.test_views.TestCreated.test_mean_dimensions ... ok
2016-12-09 11:11:45 [20161209-111132-f919] [INFO ] Job deleted.
2016-12-09 11:11:45 [20161209-111138-1645] [INFO ] Job deleted.
2016-12-09 11:11:45 [20161209-111145-8e41] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:11:45 [20161209-111145-8e41] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:11:45 [20161209-111145-8e41] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41 there: True True
Running in slurm mode
2016-12-09 11:11:45 [20161209-111145-8e41] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmpKKw3ev /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41/labels.txt --min=2 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41/val.txt --percent_val=25.0"
2016-12-09 11:11:45 [20161209-111145-8e41] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016045
2016-12-09 11:11:45 [20161209-111145-8e41] [WARNING] Parse Folder (train/val) unrecognized output: srun: Job step created
2016-12-09 11:11:47 [20161209-111145-8e41] [WARNING] Parse Folder (train/val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:47 [20161209-111145-8e41] [WARNING] Parse Folder (train/val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:47 [20161209-111145-8e41] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016045
2016-12-09 11:11:47 [20161209-111145-8e41] [INFO ] Parse Folder (train/val) task completed.
2016-12-09 11:11:47 [20161209-111145-8e41] [INFO ] Create DB (train) task started.
2016-12-09 11:11:47 [20161209-111145-8e41] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:47 [20161209-111145-8e41] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41 there: True True
Running in slurm mode
2016-12-09 11:11:47 [20161209-111145-8e41] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41/train.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41/train_db 10 10 --backend=lmdb --channels=1 --resize_mode=squash --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41/mean.binaryproto --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41/mean.jpg --shuffle --encoding=png"
2016-12-09 11:11:47 [20161209-111145-8e41] [INFO ] Create DB (val) task started.
2016-12-09 11:11:47 [20161209-111145-8e41] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:47 [20161209-111145-8e41] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41 there: True True
Running in slurm mode
2016-12-09 11:11:47 [20161209-111145-8e41] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41/val.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111145-8e41/val_db 10 10 --backend=lmdb --channels=1 --resize_mode=squash --shuffle --encoding=png"
2016-12-09 11:11:47 [20161209-111145-8e41] [WARNING] Create DB (train) unrecognized output: salloc: Granted job allocation 4016046
2016-12-09 11:11:47 [20161209-111145-8e41] [WARNING] Create DB (val) unrecognized output: salloc: Granted job allocation 4016047
2016-12-09 11:11:48 [20161209-111145-8e41] [WARNING] Create DB (train) unrecognized output: srun: Job step created
2016-12-09 11:11:48 [20161209-111145-8e41] [WARNING] Create DB (val) unrecognized output: srun: Job step created
2016-12-09 11:11:50 [20161209-111145-8e41] [WARNING] Create DB (train) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:50 [20161209-111145-8e41] [WARNING] Create DB (train) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:50 [20161209-111145-8e41] [WARNING] Create DB (val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:50 [20161209-111145-8e41] [WARNING] Create DB (val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:50 [20161209-111145-8e41] [DEBUG] 24 images written to database
2016-12-09 11:11:50 [20161209-111145-8e41] [DEBUG] 6 images written to database
2016-12-09 11:11:50 [20161209-111145-8e41] [WARNING] Create DB (train) unrecognized output: salloc: Relinquishing job allocation 4016046
2016-12-09 11:11:50 [20161209-111145-8e41] [INFO ] Create DB (train) task completed.
2016-12-09 11:11:50 [20161209-111145-8e41] [WARNING] Create DB (val) unrecognized output: salloc: Relinquishing job allocation 4016047
2016-12-09 11:11:50 [20161209-111145-8e41] [INFO ] Create DB (val) task completed.
2016-12-09 11:11:51 [20161209-111145-8e41] [INFO ] Job complete.
digits.dataset.images.classification.test_views.TestCreatedGrayscale.test_backend_selection ... ok
digits.dataset.images.classification.test_views.TestCreatedGrayscale.test_clone ... 2016-12-09 11:11:51 [20161209-111151-be83] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:11:51 [20161209-111151-be83] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:11:51 [20161209-111151-be83] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83 there: True True
Running in slurm mode
2016-12-09 11:11:51 [20161209-111151-be83] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmpKKw3ev /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83/labels.txt --min=1 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83/val.txt --percent_val=25.0 --max=3"
2016-12-09 11:11:51 [20161209-111151-be83] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016048
2016-12-09 11:11:51 [20161209-111151-be83] [WARNING] Parse Folder (train/val) unrecognized output: srun: Job step created
2016-12-09 11:11:53 [20161209-111151-be83] [WARNING] Parse Folder (train/val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:53 [20161209-111151-be83] [WARNING] Parse Folder (train/val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:53 [20161209-111151-be83] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:53 [20161209-111151-be83] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:53 [20161209-111151-be83] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:53 [20161209-111151-be83] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016048
2016-12-09 11:11:53 [20161209-111151-be83] [INFO ] Parse Folder (train/val) task completed.
2016-12-09 11:11:53 [20161209-111151-be83] [INFO ] Create DB (train) task started.
2016-12-09 11:11:53 [20161209-111151-be83] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:53 [20161209-111151-be83] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83 there: True True
Running in slurm mode
2016-12-09 11:11:53 [20161209-111151-be83] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83/train.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83/train_db 10 10 --backend=lmdb --channels=1 --resize_mode=half_crop --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83/mean.binaryproto --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83/mean.jpg --shuffle --encoding=png"
2016-12-09 11:11:53 [20161209-111151-be83] [INFO ] Create DB (val) task started.
2016-12-09 11:11:53 [20161209-111151-be83] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:53 [20161209-111151-be83] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83 there: True True
Running in slurm mode
2016-12-09 11:11:53 [20161209-111151-be83] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83/val.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111151-be83/val_db 10 10 --backend=lmdb --channels=1 --resize_mode=half_crop --shuffle --encoding=png"
2016-12-09 11:11:53 [20161209-111151-be83] [WARNING] Create DB (train) unrecognized output: salloc: Granted job allocation 4016049
2016-12-09 11:11:54 [20161209-111151-be83] [WARNING] Create DB (val) unrecognized output: salloc: Granted job allocation 4016050
2016-12-09 11:11:54 [20161209-111151-be83] [WARNING] Create DB (train) unrecognized output: srun: Job step created
2016-12-09 11:11:54 [20161209-111151-be83] [WARNING] Create DB (val) unrecognized output: srun: Job step created
2016-12-09 11:11:56 [20161209-111151-be83] [WARNING] Create DB (val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:56 [20161209-111151-be83] [WARNING] Create DB (val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:56 [20161209-111151-be83] [WARNING] Create DB (train) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:56 [20161209-111151-be83] [WARNING] Create DB (train) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:57 [20161209-111151-be83] [DEBUG] 3 images written to database
2016-12-09 11:11:57 [20161209-111151-be83] [DEBUG] 6 images written to database
2016-12-09 11:11:57 [20161209-111151-be83] [WARNING] Create DB (val) unrecognized output: salloc: Relinquishing job allocation 4016050
2016-12-09 11:11:57 [20161209-111151-be83] [INFO ] Create DB (val) task completed.
2016-12-09 11:11:57 [20161209-111151-be83] [WARNING] Create DB (train) unrecognized output: salloc: Relinquishing job allocation 4016049
2016-12-09 11:11:57 [20161209-111151-be83] [INFO ] Create DB (train) task completed.
2016-12-09 11:11:57 [20161209-111151-be83] [INFO ] Job complete.
2016-12-09 11:11:57 [20161209-111157-eb17] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:11:57 [20161209-111157-eb17] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:11:57 [20161209-111157-eb17] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17 there: True True
Running in slurm mode
2016-12-09 11:11:57 [20161209-111157-eb17] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmpKKw3ev /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17/labels.txt --min=1 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17/val.txt --percent_val=25.0 --max=3"
2016-12-09 11:11:57 [20161209-111157-eb17] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016051
2016-12-09 11:11:57 [20161209-111157-eb17] [WARNING] Parse Folder (train/val) unrecognized output: srun: Job step created
2016-12-09 11:11:59 [20161209-111157-eb17] [WARNING] Parse Folder (train/val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:11:59 [20161209-111157-eb17] [WARNING] Parse Folder (train/val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:11:59 [20161209-111157-eb17] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:59 [20161209-111157-eb17] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:59 [20161209-111157-eb17] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:11:59 [20161209-111157-eb17] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016051
2016-12-09 11:11:59 [20161209-111157-eb17] [INFO ] Parse Folder (train/val) task completed.
2016-12-09 11:11:59 [20161209-111157-eb17] [INFO ] Create DB (train) task started.
2016-12-09 11:11:59 [20161209-111157-eb17] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:59 [20161209-111157-eb17] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17 there: True True
Running in slurm mode
2016-12-09 11:11:59 [20161209-111157-eb17] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17/train.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17/train_db 10 10 --backend=lmdb --channels=1 --resize_mode=half_crop --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17/mean.binaryproto --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17/mean.jpg --shuffle --encoding=png"
2016-12-09 11:11:59 [20161209-111157-eb17] [INFO ] Create DB (val) task started.
2016-12-09 11:11:59 [20161209-111157-eb17] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:11:59 [20161209-111157-eb17] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17 there: True True
Running in slurm mode
2016-12-09 11:11:59 [20161209-111157-eb17] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17/val.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111157-eb17/val_db 10 10 --backend=lmdb --channels=1 --resize_mode=half_crop --shuffle --encoding=png"
2016-12-09 11:11:59 [20161209-111157-eb17] [WARNING] Create DB (train) unrecognized output: salloc: Granted job allocation 4016052
2016-12-09 11:12:00 [20161209-111157-eb17] [WARNING] Create DB (val) unrecognized output: salloc: Granted job allocation 4016053
2016-12-09 11:12:00 [20161209-111157-eb17] [WARNING] Create DB (train) unrecognized output: srun: Job step created
2016-12-09 11:12:00 [20161209-111157-eb17] [WARNING] Create DB (val) unrecognized output: srun: Job step created
2016-12-09 11:12:02 [20161209-111157-eb17] [WARNING] Create DB (train) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:02 [20161209-111157-eb17] [WARNING] Create DB (train) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:02 [20161209-111157-eb17] [WARNING] Create DB (val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:02 [20161209-111157-eb17] [WARNING] Create DB (val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:03 [20161209-111157-eb17] [DEBUG] 6 images written to database
2016-12-09 11:12:03 [20161209-111157-eb17] [WARNING] Create DB (train) unrecognized output: salloc: Relinquishing job allocation 4016052
2016-12-09 11:12:03 [20161209-111157-eb17] [DEBUG] 3 images written to database
2016-12-09 11:12:03 [20161209-111157-eb17] [WARNING] Create DB (val) unrecognized output: salloc: Relinquishing job allocation 4016053
2016-12-09 11:12:04 [20161209-111157-eb17] [WARNING] Create DB (train) unrecognized output: salloc: Job allocation 4016052 has been revoked.
2016-12-09 11:12:04 [20161209-111157-eb17] [INFO ] Create DB (train) task completed.
2016-12-09 11:12:04 [20161209-111157-eb17] [INFO ] Create DB (val) task completed.
2016-12-09 11:12:04 [20161209-111157-eb17] [INFO ] Job complete.
ok
digits.dataset.images.classification.test_views.TestCreatedGrayscale.test_dataset_json ... ok
digits.dataset.images.classification.test_views.TestCreatedGrayscale.test_edit_name ... 2016-12-09 11:12:04 [20161209-111145-8e41] [INFO ] Set name to "new name".
ok
digits.dataset.images.classification.test_views.TestCreatedGrayscale.test_edit_notes ... 2016-12-09 11:12:04 [20161209-111145-8e41] [INFO ] Updated notes.
ok
digits.dataset.images.classification.test_views.TestCreatedGrayscale.test_explore_train ... ok
digits.dataset.images.classification.test_views.TestCreatedGrayscale.test_explore_val ... ok
digits.dataset.images.classification.test_views.TestCreatedGrayscale.test_index_json ... ok
digits.dataset.images.classification.test_views.TestCreatedGrayscale.test_mean_dimensions ... ok
2016-12-09 11:12:04 [20161209-111151-be83] [INFO ] Job deleted.
2016-12-09 11:12:04 [20161209-111157-eb17] [INFO ] Job deleted.
2016-12-09 11:12:04 [20161209-111204-6a49] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:12:04 [20161209-111204-6a49] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:12:04 [20161209-111204-6a49] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49 there: True True
Running in slurm mode
2016-12-09 11:12:04 [20161209-111204-6a49] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmputUC1N /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49/labels.txt --min=2 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49/val.txt --percent_val=25.0"
2016-12-09 11:12:04 [20161209-111204-6a49] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016054
2016-12-09 11:12:06 [20161209-111204-6a49] [WARNING] Parse Folder (train/val) unrecognized output: srun: Job step created
2016-12-09 11:12:08 [20161209-111204-6a49] [WARNING] Parse Folder (train/val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:08 [20161209-111204-6a49] [WARNING] Parse Folder (train/val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:08 [20161209-111204-6a49] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016054
2016-12-09 11:12:09 [20161209-111204-6a49] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Job allocation 4016054 has been revoked.
2016-12-09 11:12:09 [20161209-111204-6a49] [INFO ] Parse Folder (train/val) task completed.
2016-12-09 11:12:09 [20161209-111204-6a49] [INFO ] Create DB (train) task started.
2016-12-09 11:12:09 [20161209-111204-6a49] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:12:09 [20161209-111204-6a49] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49 there: True True
Running in slurm mode
2016-12-09 11:12:09 [20161209-111204-6a49] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49/train.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49/train_db 10 10 --backend=hdf5 --channels=3 --resize_mode=squash --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49/mean.binaryproto --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49/mean.jpg --shuffle --hdf5_dset_limit=2147483648"
2016-12-09 11:12:09 [20161209-111204-6a49] [INFO ] Create DB (val) task started.
2016-12-09 11:12:09 [20161209-111204-6a49] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:12:09 [20161209-111204-6a49] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49 there: True True
Running in slurm mode
2016-12-09 11:12:09 [20161209-111204-6a49] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49/val.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111204-6a49/val_db 10 10 --backend=hdf5 --channels=3 --resize_mode=squash --shuffle --hdf5_dset_limit=2147483648"
2016-12-09 11:12:09 [20161209-111204-6a49] [WARNING] Create DB (train) unrecognized output: salloc: Granted job allocation 4016055
2016-12-09 11:12:09 [20161209-111204-6a49] [WARNING] Create DB (val) unrecognized output: salloc: Granted job allocation 4016056
2016-12-09 11:12:09 [20161209-111204-6a49] [WARNING] Create DB (train) unrecognized output: srun: Job step created
2016-12-09 11:12:09 [20161209-111204-6a49] [WARNING] Create DB (val) unrecognized output: srun: Job step created
2016-12-09 11:12:12 [20161209-111204-6a49] [WARNING] Create DB (train) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:12 [20161209-111204-6a49] [WARNING] Create DB (train) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:12 [20161209-111204-6a49] [WARNING] Create DB (val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:12 [20161209-111204-6a49] [WARNING] Create DB (val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:12 [20161209-111204-6a49] [DEBUG] 24 images written to database
2016-12-09 11:12:12 [20161209-111204-6a49] [DEBUG] 6 images written to database
2016-12-09 11:12:12 [20161209-111204-6a49] [WARNING] Create DB (train) unrecognized output: salloc: Relinquishing job allocation 4016055
2016-12-09 11:12:12 [20161209-111204-6a49] [WARNING] Create DB (val) unrecognized output: salloc: Relinquishing job allocation 4016056
2016-12-09 11:12:14 [20161209-111204-6a49] [INFO ] Create DB (train) task completed.
2016-12-09 11:12:14 [20161209-111204-6a49] [INFO ] Create DB (val) task completed.
2016-12-09 11:12:14 [20161209-111204-6a49] [INFO ] Job complete.
digits.dataset.images.classification.test_views.TestCreatedHdf5.test_backend_selection ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5.test_clone ... 2016-12-09 11:12:14 [20161209-111214-7a5f] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:12:14 [20161209-111214-7a5f] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:12:14 [20161209-111214-7a5f] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f there: True True
Running in slurm mode
2016-12-09 11:12:14 [20161209-111214-7a5f] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmputUC1N /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f/labels.txt --min=1 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f/val.txt --percent_val=25.0 --max=3"
2016-12-09 11:12:14 [20161209-111214-7a5f] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016057
2016-12-09 11:12:18 [20161209-111214-7a5f] [WARNING] Parse Folder (train/val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:18 [20161209-111214-7a5f] [WARNING] Parse Folder (train/val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:18 [20161209-111214-7a5f] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:12:18 [20161209-111214-7a5f] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:12:18 [20161209-111214-7a5f] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:12:18 [20161209-111214-7a5f] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016057
2016-12-09 11:12:19 [20161209-111214-7a5f] [INFO ] Parse Folder (train/val) task completed.
2016-12-09 11:12:19 [20161209-111214-7a5f] [INFO ] Create DB (train) task started.
2016-12-09 11:12:19 [20161209-111214-7a5f] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:12:19 [20161209-111214-7a5f] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f there: True True
Running in slurm mode
2016-12-09 11:12:19 [20161209-111214-7a5f] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f/train.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f/train_db 10 10 --backend=hdf5 --channels=3 --resize_mode=half_crop --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f/mean.binaryproto --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f/mean.jpg --shuffle --hdf5_dset_limit=2147483648"
2016-12-09 11:12:19 [20161209-111214-7a5f] [INFO ] Create DB (val) task started.
2016-12-09 11:12:19 [20161209-111214-7a5f] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:12:19 [20161209-111214-7a5f] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f there: True True
Running in slurm mode
2016-12-09 11:12:19 [20161209-111214-7a5f] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f/val.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111214-7a5f/val_db 10 10 --backend=hdf5 --channels=3 --resize_mode=half_crop --shuffle --hdf5_dset_limit=2147483648"
2016-12-09 11:12:19 [20161209-111214-7a5f] [WARNING] Create DB (train) unrecognized output: salloc: Granted job allocation 4016058
2016-12-09 11:12:19 [20161209-111214-7a5f] [WARNING] Create DB (val) unrecognized output: salloc: Granted job allocation 4016059
2016-12-09 11:12:19 [20161209-111214-7a5f] [WARNING] Create DB (train) unrecognized output: srun: Job step created
2016-12-09 11:12:19 [20161209-111214-7a5f] [WARNING] Create DB (val) unrecognized output: srun: Job step created
2016-12-09 11:12:22 [20161209-111214-7a5f] [WARNING] Create DB (train) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:22 [20161209-111214-7a5f] [WARNING] Create DB (train) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:22 [20161209-111214-7a5f] [WARNING] Create DB (val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:22 [20161209-111214-7a5f] [WARNING] Create DB (val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:22 [20161209-111214-7a5f] [DEBUG] 6 images written to database
2016-12-09 11:12:22 [20161209-111214-7a5f] [DEBUG] 3 images written to database
2016-12-09 11:12:22 [20161209-111214-7a5f] [WARNING] Create DB (train) unrecognized output: salloc: Relinquishing job allocation 4016058
2016-12-09 11:12:22 [20161209-111214-7a5f] [WARNING] Create DB (val) unrecognized output: salloc: Relinquishing job allocation 4016059
2016-12-09 11:12:24 [20161209-111214-7a5f] [WARNING] Create DB (train) unrecognized output: salloc: Job allocation 4016058 has been revoked.
2016-12-09 11:12:24 [20161209-111214-7a5f] [INFO ] Create DB (train) task completed.
2016-12-09 11:12:24 [20161209-111214-7a5f] [INFO ] Create DB (val) task completed.
2016-12-09 11:12:24 [20161209-111214-7a5f] [INFO ] Job complete.
2016-12-09 11:12:24 [20161209-111224-f262] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:12:24 [20161209-111224-f262] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:12:24 [20161209-111224-f262] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262 there: True True
Running in slurm mode
2016-12-09 11:12:24 [20161209-111224-f262] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmputUC1N /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262/labels.txt --min=1 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262/val.txt --percent_val=25.0 --max=3"
2016-12-09 11:12:25 [20161209-111224-f262] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016060
2016-12-09 11:12:29 [20161209-111224-f262] [WARNING] Parse Folder (train/val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:29 [20161209-111224-f262] [WARNING] Parse Folder (train/val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:29 [20161209-111224-f262] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:12:29 [20161209-111224-f262] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:12:29 [20161209-111224-f262] [WARNING] Parse Folder (train/val): Reached maximum limit for this category
2016-12-09 11:12:29 [20161209-111224-f262] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016060
2016-12-09 11:12:29 [20161209-111224-f262] [INFO ] Parse Folder (train/val) task completed.
2016-12-09 11:12:29 [20161209-111224-f262] [INFO ] Create DB (train) task started.
2016-12-09 11:12:29 [20161209-111224-f262] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:12:29 [20161209-111224-f262] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262 there: True True
Running in slurm mode
2016-12-09 11:12:29 [20161209-111224-f262] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262/train.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262/train_db 10 10 --backend=hdf5 --channels=3 --resize_mode=half_crop --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262/mean.binaryproto --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262/mean.jpg --shuffle --hdf5_dset_limit=2147483648"
2016-12-09 11:12:29 [20161209-111224-f262] [INFO ] Create DB (val) task started.
2016-12-09 11:12:29 [20161209-111224-f262] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:12:29 [20161209-111224-f262] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262 there: True True
Running in slurm mode
2016-12-09 11:12:29 [20161209-111224-f262] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262/val.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111224-f262/val_db 10 10 --backend=hdf5 --channels=3 --resize_mode=half_crop --shuffle --hdf5_dset_limit=2147483648"
2016-12-09 11:12:29 [20161209-111224-f262] [WARNING] Create DB (train) unrecognized output: salloc: Granted job allocation 4016061
2016-12-09 11:12:29 [20161209-111224-f262] [WARNING] Create DB (val) unrecognized output: salloc: Granted job allocation 4016062
2016-12-09 11:12:30 [20161209-111224-f262] [WARNING] Create DB (train) unrecognized output: srun: Job step created
2016-12-09 11:12:30 [20161209-111224-f262] [WARNING] Create DB (val) unrecognized output: srun: Job step created
2016-12-09 11:12:32 [20161209-111224-f262] [WARNING] Create DB (train) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:32 [20161209-111224-f262] [WARNING] Create DB (train) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:32 [20161209-111224-f262] [WARNING] Create DB (val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:32 [20161209-111224-f262] [WARNING] Create DB (val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:32 [20161209-111224-f262] [DEBUG] 6 images written to database
2016-12-09 11:12:32 [20161209-111224-f262] [WARNING] Create DB (train) unrecognized output: salloc: Relinquishing job allocation 4016061
2016-12-09 11:12:32 [20161209-111224-f262] [DEBUG] 3 images written to database
2016-12-09 11:12:33 [20161209-111224-f262] [WARNING] Create DB (val) unrecognized output: salloc: Relinquishing job allocation 4016062
2016-12-09 11:12:34 [20161209-111224-f262] [INFO ] Create DB (train) task completed.
2016-12-09 11:12:34 [20161209-111224-f262] [WARNING] Create DB (val) unrecognized output: salloc: Job allocation 4016062 has been revoked.
2016-12-09 11:12:34 [20161209-111224-f262] [INFO ] Create DB (val) task completed.
2016-12-09 11:12:34 [20161209-111224-f262] [INFO ] Job complete.
ok
digits.dataset.images.classification.test_views.TestCreatedHdf5.test_compression_method ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5.test_dataset_json ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5.test_edit_name ... 2016-12-09 11:12:35 [20161209-111204-6a49] [INFO ] Set name to "new name".
ok
digits.dataset.images.classification.test_views.TestCreatedHdf5.test_edit_notes ... 2016-12-09 11:12:35 [20161209-111204-6a49] [INFO ] Updated notes.
ok
digits.dataset.images.classification.test_views.TestCreatedHdf5.test_explore_train ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5.test_explore_val ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5.test_index_json ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5.test_mean_dimensions ... ok
2016-12-09 11:12:35 [20161209-111214-7a5f] [INFO ] Job deleted.
2016-12-09 11:12:35 [20161209-111224-f262] [INFO ] Job deleted.
2016-12-09 11:12:35 [20161209-111235-50f4] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:12:35 [20161209-111235-50f4] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:12:35 [20161209-111235-50f4] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4 there: True True
Running in slurm mode
2016-12-09 11:12:35 [20161209-111235-50f4] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmpAe_7K6 /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4/labels.txt --min=2 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4/val.txt --percent_val=25.0"
2016-12-09 11:12:37 [20161209-111235-50f4] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016063
2016-12-09 11:12:37 [20161209-111235-50f4] [WARNING] Parse Folder (train/val) unrecognized output: srun: Job step created
2016-12-09 11:12:39 [20161209-111235-50f4] [WARNING] Parse Folder (train/val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:39 [20161209-111235-50f4] [WARNING] Parse Folder (train/val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:39 [20161209-111235-50f4] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016063
2016-12-09 11:12:39 [20161209-111235-50f4] [INFO ] Parse Folder (train/val) task completed.
2016-12-09 11:12:39 [20161209-111235-50f4] [INFO ] Create DB (train) task started.
2016-12-09 11:12:39 [20161209-111235-50f4] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:12:39 [20161209-111235-50f4] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4 there: True True
Running in slurm mode
2016-12-09 11:12:39 [20161209-111235-50f4] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4/train.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4/train_db 10 10 --backend=hdf5 --channels=3 --resize_mode=squash --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4/mean.binaryproto --mean_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4/mean.jpg --shuffle --compression=gzip --hdf5_dset_limit=2147483648"
2016-12-09 11:12:39 [20161209-111235-50f4] [INFO ] Create DB (val) task started.
2016-12-09 11:12:39 [20161209-111235-50f4] [INFO ] <class 'digits.dataset.tasks.create_db.CreateDbTask'>
2016-12-09 11:12:39 [20161209-111235-50f4] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4 there: True True
Running in slurm mode
2016-12-09 11:12:39 [20161209-111235-50f4] [INFO ] Task subprocess args: "salloc -t 30 -c 8 --mem=10GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/create_db.py /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4/val.txt /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111235-50f4/val_db 10 10 --backend=hdf5 --channels=3 --resize_mode=squash --shuffle --compression=gzip --hdf5_dset_limit=2147483648"
2016-12-09 11:12:39 [20161209-111235-50f4] [WARNING] Create DB (train) unrecognized output: salloc: Granted job allocation 4016064
2016-12-09 11:12:39 [20161209-111235-50f4] [WARNING] Create DB (val) unrecognized output: salloc: Granted job allocation 4016065
2016-12-09 11:12:42 [20161209-111235-50f4] [WARNING] Create DB (train) unrecognized output: srun: Job step created
2016-12-09 11:12:42 [20161209-111235-50f4] [WARNING] Create DB (val) unrecognized output: srun: Job step created
2016-12-09 11:12:44 [20161209-111235-50f4] [WARNING] Create DB (train) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:44 [20161209-111235-50f4] [WARNING] Create DB (train) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:44 [20161209-111235-50f4] [WARNING] Create DB (val) unrecognized output: cudaRuntimeGetVersion() failed with error #38
2016-12-09 11:12:44 [20161209-111235-50f4] [WARNING] Create DB (val) unrecognized output: WARNING: log_file config option not found - no log file is being saved
2016-12-09 11:12:44 [20161209-111235-50f4] [DEBUG] 24 images written to database
2016-12-09 11:12:45 [20161209-111235-50f4] [WARNING] Create DB (train) unrecognized output: salloc: Relinquishing job allocation 4016064
2016-12-09 11:12:45 [20161209-111235-50f4] [INFO ] Create DB (train) task completed.
2016-12-09 11:12:45 [20161209-111235-50f4] [DEBUG] 6 images written to database
2016-12-09 11:12:45 [20161209-111235-50f4] [WARNING] Create DB (val) unrecognized output: salloc: Relinquishing job allocation 4016065
2016-12-09 11:12:47 [20161209-111235-50f4] [INFO ] Create DB (val) task completed.
2016-12-09 11:12:47 [20161209-111235-50f4] [INFO ] Job complete.
digits.dataset.images.classification.test_views.TestCreatedHdf5Gzip.test_backend_selection ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5Gzip.test_clone ... 2016-12-09 11:12:47 [20161209-111247-d865] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:12:47 [20161209-111247-d865] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:12:47 [20161209-111247-d865] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111247-d865 there: True True
Running in slurm mode
2016-12-09 11:12:47 [20161209-111247-d865] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmpAe_7K6 /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111247-d865/labels.txt --min=1 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111247-d865/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111247-d865/val.txt --percent_val=25.0 --max=3"
2016-12-09 11:12:49 [20161209-111247-d865] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016066
2016-12-09 11:12:50 [20161209-111247-d865] [WARNING] Parse Folder (train/val) unrecognized output: srun: Cancelled pending job step
2016-12-09 11:12:50 [20161209-111247-d865] [WARNING] Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016066
2016-12-09 11:12:50 [20161209-111247-d865] [ERROR] TypeError: not enough arguments for format string
FAIL
digits.dataset.images.classification.test_views.TestCreatedHdf5Gzip.test_compression_method ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5Gzip.test_dataset_json ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5Gzip.test_edit_name ... 2016-12-09 11:12:50 [20161209-111235-50f4] [INFO ] Set name to "new name".
ok
digits.dataset.images.classification.test_views.TestCreatedHdf5Gzip.test_edit_notes ... 2016-12-09 11:12:50 [20161209-111235-50f4] [INFO ] Updated notes.
ok
digits.dataset.images.classification.test_views.TestCreatedHdf5Gzip.test_explore_train ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5Gzip.test_explore_val ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5Gzip.test_index_json ... ok
digits.dataset.images.classification.test_views.TestCreatedHdf5Gzip.test_mean_dimensions ... ok
2016-12-09 11:12:50 [20161209-111247-d865] [INFO ] Job deleted.
2016-12-09 11:12:50 [20161209-111250-cfe2] [INFO ] Parse Folder (train/val) task started.
2016-12-09 11:12:50 [20161209-111250-cfe2] [INFO ] <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
2016-12-09 11:12:50 [20161209-111250-cfe2] [INFO ] is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111250-cfe2 there: True True
Running in slurm mode
2016-12-09 11:12:50 [20161209-111250-cfe2] [INFO ] Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmp6DzTT2 /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111250-cfe2/labels.txt --min=2 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111250-cfe2/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111250-cfe2/val.txt --percent_val=25.0"
KeyboardInterrupt

======================================================================
FAIL: digits.dataset.generic.test_views.TestImageGradientViews.test_page_dataset_new
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/apps/python/2.7.11/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/data/zim021/digits-proj/DIGITS/digits/dataset/generic/test_views.py", line 205, in test_page_dataset_new
    assert rv.status_code == 200, 'page load failed with %s' % rv.status_code
AssertionError: page load failed with 500

======================================================================
FAIL: digits.dataset.images.classification.test_views.TestCreatedHdf5Gzip.test_clone
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/apps/python/2.7.11/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/data/zim021/digits-proj/DIGITS/digits/dataset/images/classification/test_views.py", line 183, in test_clone
    assert self.dataset_wait_completion(job1_id) == 'Done', 'first job failed'
AssertionError: first job failed
-------------------- >> begin captured logging << --------------------
socketio: INFO: emitting event "job update" to job_management [/jobs]
socketio: INFO: emitting event "job update" to 20161209-111247-d865 [/jobs]
socketio: INFO: emitting event "job update" to job_management [/jobs]
digits.webapp: INFO: Parse Folder (train/val) task started.
socketio: INFO: emitting event "task update" to 20161209-111247-d865 [/jobs]
socketio: INFO: emitting event "job update" to 20161209-111247-d865 [/jobs]
socketio: INFO: emitting event "job update" to job_management [/jobs]
digits.webapp: INFO: <class 'digits.dataset.tasks.parse_folder.ParseFolderTask'>
digits.webapp: INFO: is /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111247-d865 there: True True
digits.webapp: INFO: Task subprocess args: "salloc -t 30 -c 4 --mem=8GB --gres=gpu:0 srun /data/zim021/digits-env/bin/python2 /data/zim021/digits-proj/DIGITS/digits/tools/parse_folder.py /data/zim021/digits-proj/tmp/tmpAe_7K6 /data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111247-d865/labels.txt --min=1 --train_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111247-d865/train.txt --val_file=/data/zim021/digits-proj/tmp/tmpfEJdl5/20161209-111247-d865/val.txt --percent_val=25.0 --max=3"
digits.webapp: WARNING: Parse Folder (train/val) unrecognized output: salloc: Granted job allocation 4016066
digits.webapp: WARNING: Parse Folder (train/val) unrecognized output: srun: Cancelled pending job step
digits.webapp: WARNING: Parse Folder (train/val) unrecognized output: salloc: Relinquishing job allocation 4016066
digits: ERROR: TypeError: not enough arguments for format string
socketio: INFO: emitting event "task update" to 20161209-111247-d865 [/jobs]
socketio: INFO: emitting event "job update" to 20161209-111247-d865 [/jobs]
socketio: INFO: emitting event "job update" to job_management [/jobs]
socketio: INFO: emitting event "server update" to job_management [/jobs]
socketio: INFO: emitting event "job update" to 20161209-111247-d865 [/jobs]
socketio: INFO: emitting event "job update" to job_management [/jobs]
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 54 tests in 95.433s

FAILED (SKIP=2, failures=2)
